js/profile-geonovum.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+// In case everything else fails, we want the error
+window.addEventListener("error", ev => {
+  console.error(ev.error, ev.message, ev);
+});
+
+// this is only set in a build, not at all in the dev environment
+require.config({
```

Comment:
Take a look at the current w3c profile about this configuration... we've managed to reduce it quite a bit in the w3c profile, so you might not need most of this.

Reply:
Ooops! sorry, I was wrong.

Reply:
I took the w3c profile (from develop) as an example. I could check what we need from it.